### PR TITLE
add r-kendall

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by FernandezMathieu** [comment]
@conda-forge-linter restart ci

**Comment by FernandezMathieu** [comment]
Hello @dbast did you have any idea why i can't build on windows ?

**Comment by FernandezMathieu** [comment]
@conda-forge-admin did you have any idea why i can't build on windows ?

Kendall throw one error in build :
mingw32-make: *** No rule to make target '$(FCLIBS_XTRA)''. Stop.

**Comment by dbast** [comment]
Browsing the source tells that this is not noarch, but requires fortran compilers.

File: recipes/r-kendall/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - /lib64/libm.so.6
+    - /lib64/libpthread.so.0
+    - /lib64/libc.so.6
```
Comment (dbast) [inline recipes/r-kendall/meta.yaml:24]: I think those can be removed:
```suggestion
```

**Comment by FernandezMathieu** [comment]
Thank you so much @dbast

**Comment by dbast** [comment]
green now :)

**Pull request merged**